pkg/features/list.go

```diff
@@ -44,4 +44,7 @@ var (
 
 	// ComplianceEnhancements enables APIs and UI pages for Compliance 2.0
 	ComplianceEnhancements = registerFeature("Compliance enhancements", "ROX_COMPLIANCE_ENHANCEMENTS", false)
+
+	// ExposeCentralEvents enables APIs and UI pages for exposing Central events.
+	ExposeCentralEvents = registerFeature("Expose Central events", "ROX_CENTRAL_EVENTS", false)
```

Comment:
So the feature flag only gates the API being exposed, but in the background the events are still being collected and persisted? Shouldn't we have a feature flag for the entire central events backend?

Reply:
Looking at the other examples, this is what we generally write as generic comment when adding feature flags.

When it's disable, we won't do anything on the backend (nothing will be persisted nor intercepted), I can change the comment if you want.

Reply:
Maybe `s/Expose/Enable`?